pkg/probe/wifi_managed_ap.go

```diff
@@ -0,0 +1,240 @@
+			"total number of packages received on this interface",
+			[]string{"vdom", "ap_name", "interface"}, nil,
+		)
+		interfacePackagesTx = prometheus.NewDesc(
+			"fortigate_wifi_managed_ap_interface_tx_packages_total",
+			"total number of packages transferred on this interface",
+			[]string{"vdom", "ap_name", "interface"}, nil,
+		)
+		interfaceErrorsRx = prometheus.NewDesc(
+			"fortigate_wifi_managed_ap_interface_rx_errors_total",
+			"total number of errors received on this interface",
+			[]string{"vdom", "ap_name", "interface"}, nil,
+		)
+		interfaceErrorsTx = prometheus.NewDesc(
+			"fortigate_wifi_managed_ap_interface_tx_errors_total",
+			"total number of errors transferred on this interface",
+			[]string{"vdom", "ap_name", "interface"}, nil,
+		)
+		interfaceDroppedRx = prometheus.NewDesc(
+			"fortigate_wifi_managed_ap_interface_rx_dropped_packages_total",
+			"total number of dropped packages received on this interface",
+			[]string{"vdom", "ap_name", "interface"}, nil,
+		)
+		interfaceDroppedTx = prometheus.NewDesc(
+			"fortigate_wifi_managed_ap_interface_tx_dropped_packages_total",
+			"total number of dropped packages transferred on this interface",
+			[]string{"vdom", "ap_name", "interface"}, nil,
+		)
+	)
+
+	type Radio struct {
+		RadioID                   int     `json:"radio_id"`
+		ClientCount               float64 `json:"client_count,omitempty"`
+		OperChan                  int     `json:"oper_chan,omitempty"`
+		OperTxpower               float64 `json:"oper_txpower,omitempty"`
+		ChannelUtilizationPercent float64 `json:"channel_utilization_percent,omitempty"`
+		BandwidthRx               float64 `json:"bandwidth_rx,omitempty"`
+		BandwidthTx               float64 `json:"bandwidth_tx,omitempty"`
+		BytesRx                   float64 `json:"bytes_rx,omitempty"`
+		BytesTx                   float64 `json:"bytes_tx,omitempty"`
+		InterferingAps            float64 `json:"interfering_aps,omitempty"`
+		Txpower                   float64 `json:"txpower,omitempty"`
+		TxRetriesPercent          float64 `json:"tx_retries_percent,omitempty"`
+		TxDiscardPercentage       float64 `json:"tx_discard_percentage,omitempty"`
+	}
+
+	type Wired struct {
+		Interface  string  `json:"interface"`
+		BytesRx    float64 `json:"bytes_rx"`
+		BytesTx    float64 `json:"bytes_tx"`
+		PacketsRx  float64 `json:"packets_rx"`
+		PacketsTx  float64 `json:"packets_tx"`
+		ErrorsRx   float64 `json:"errors_rx"`
+		ErrorsTx   float64 `json:"errors_tx"`
+		DroppedRx  float64 `json:"dropped_rx"`
+		DroppedTx  float64 `json:"dropped_tx"`
+		Collisions float64 `json:"collisions"`
+	}
+	type WanStatus struct {
+		Interface     string `json:"interface"`
+		LinkSpeedMbps int    `json:"link_speed_mbps"`
+		CarrierLink   bool   `json:"carrier_link"`
+		FullDuplex    bool   `json:"full_duplex"`
+	}
+
+	type Results struct {
+		Name        string      `json:"name"`
+		Vdom        string      `json:"vdom"`
```

Comment:
```suggestion
		VDOM        string      `json:"vdom"`
```

Reply:
yeah I auto generate this stuff, that's the reason why this comes out like this. I will change that and the others.

Reply:
Sure, It's a stupid Go-rule, I don't judge. Would have been easier if they chose to treat all words the same - oh well, rules are rules.